scripts/image

```diff
@@ -47,7 +47,11 @@ ORIGIN_URL="$(git remote -v | awk '$1 == "origin" { print $2 }' | head -1 | sed
 if [ "$OFFICIAL" = "yes" ]; then
   LIBREELEC_BUILD="official"
 else
-  LIBREELEC_BUILD="community"
+  if [ -z "$BUILDER_NAME" ] ; then
+    LIBREELEC_BUILD="community"
+  else
+    LIBREELEC_BUILD="$BUILDER_NAME"
+  fi
```

Comment:
```
if [ -n "$BUILDER_NAME" ]; then
  LIBREELEC_BUILD="$BUILDER_NAME"
else
  LIBREELEC_BUILD="community"
fi
```

Is more straightforward in tying what it's testing for to what it's setting it to.